services/social-groups-service/src/main/resources/db/migration/V9__account_friend_links_reciprocal_lookup_index.sql

```
@@ -0,0 +1,2 @@
+CREATE INDEX idx_account_friend_links_reciprocal_lookup
+    ON account_friend_links(tenant_id, friend_account_id, account_id, status);
```

SQLFluff (4.2.2)

[error] CREATE INDEX should use CONCURRENTLY to avoid locking the table during the build.

(PG01)